lib/fluent/plugin/base.rb

```diff
@@ -51,6 +51,12 @@ def fluentd_worker_id
         @_fluentd_worker_id
       end
 
+      def fluentd_lockdir
+        return @_fluentd_lockdir if @_fluentd_lockdir
+        @_fluentd_lockdir = ENV['FLUENTD_LOCKDIR']
+        @_fluentd_lockdir
```

Comment:
We can simplify this:

```suggestion
        @_fluentd_lockdir ||= ENV['FLUENTD_LOCKDIR']
```

Reply:
https://github.com/fluent/fluentd/pull/3808/files#diff-a011588d74f931c0c2154891220af14f9181f28eb70b978d2e017866dfba3081R31-R38

I don't know if this is really necessary, but should we set nil to `@_fluentd_lockdir` here?

Reply:
Putting `@_fluentd_lockdir = ENV['FLUENTD_LOCKDIR']` into `#initialize` is better.

Reply:
Thank you for your review. Fixed by a538b9bcc6561aac500823f1b7332edae8c2d0cb